mobile/Components/ArtworkCard.js

```diff
@@ -0,0 +1,46 @@
+// component that renders artwork cards on gallery screen
+import * as React from 'react';
+import {
+  View, Image, TouchableOpacity, ScrollView,
+} from 'react-native';
+import { Card, Title, Text, Button } from 'react-native-paper';
+import PropTypes from 'prop-types';
+
+function ArtworkCard({ title, Encoding, key, name, description, email, navigation, }) {
+  const onPressEvent = () => {
+    navigation.navigate('Artwork Details', {
+      title,
+      Encoding,
+      key,
+      name,
+      description,
+      email,
+    });
+  };
+
+  return (
+    <TouchableOpacity>
+      <Card onPress={onPressEvent}>
+        <Image // works with card.cover as well
```

Comment:
Remove comment?

Reply:
want to keep cuz i want to know if it can be implemented with cover